pyproject.toml

```diff
@@ -12,6 +12,7 @@ searchconsole = "^0.0.4"
 google-cloud-bigquery = "<2.8.0"
 pandas = "^1.2.3"
 pyarrow = "^3.0.0"
+python-fb-page-insights-client = {git = "git@github.com:pycontw/python-fb-page-insights-client.git", rev = "poetry3"}
```

Comment:
If we need another pyproject, maybe let us create another repository for this codebase?